harness/determined/common/storage/azure_client.py

```diff
@@ -0,0 +1,79 @@
+import logging
+from pathlib import Path
+from typing import List, Optional, Union
+
+from azure.core.exceptions import HttpResponseError, ResourceExistsError
+from azure.storage.blob import BlobServiceClient
+from azure.storage.blob import BlobAnalyticsLogging
```

Comment:
`BlobAnalyticsLogging` seems unused?

Reply:
yup - I used it while testing how to get rid of the HTTP logging. It needs to be removed - I'll remove + lint.